sway/commands/exit.c

```diff
@@ -2,8 +2,6 @@
 #include "sway/commands.h"
 #include "sway/config.h"
```

Comment:
There's a missing include for `sway/server.h`.

Reply:
Oops, fixed!